lib/checks/mobile/meta-viewport-scale-evaluate.js

```diff
@@ -14,7 +14,7 @@ function metaViewportScaleEvaluate(node, options) {
 
 		const [key, value] = contentValue.split('=');
 		const curatedKey = key.toLowerCase().trim();
```

Comment:
Rather than creating a map (result) of key's with empty values, we should exit early when either key or value is `undefined` or `null` or has no length.

```
if(!key || !value) {
	return out
}
```

Reply:
Made that change now.